modules/core/src/v2/coins/eth.ts

```diff
@@ -224,11 +235,30 @@ interface FeeEstimate {
   feeEstimate: number;
 }
 
-interface TransactionPrebuild extends BaseTransactionPrebuild {
+export interface TransactionPrebuild extends BaseTransactionPrebuild {
   hopTransaction?: HopPrebuild;
   buildParams: {
     recipients: Recipient[];
   };
+  recipients?: TransactionRecipient[];
+  nextContractSequenceId?: string;
+  gasPrice?: number;
```

Comment:
will this field ever be empty? Unsure, if it can be ignore

Reply:
Some of these fields won't ever be empty so I updated the interface to no longer mark them as optional.